### PR TITLE
add get command for releases

### PR DESCRIPTION
**Pull request by AverageMarcus**
When creating a pull request for `kubectl-gs`, please consider:

- Provide a link to the issue, and please describe the goal you are trying to accomplish in this description.
- SIG UX cares about almost all changes here and is always happy to offer feedback. Simply request a review.
- Add a user-friendly description of your change to `CHANGELOG.md`.
- Our public [kubectl-gs documentation](https://docs.giantswarm.io/ui-api/kubectl-gs/) may need an update to reflect the change you are making. - https://github.com/giantswarm/docs/pull/1136

### CODE REVIEW

File: cmd/get/releases/command.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/giantswarm/kubectl-gs/pkg/middleware"
+	"github.com/giantswarm/kubectl-gs/pkg/middleware/renewtoken"
+)
+
+const (
+	name  = "releases [release-name]"
```
Comment (axbarsan) [inline cmd/get/releases/command.go:18]: Why is this called a `name` and not a `version`?

File: cmd/get/releases/command.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	alias = "release"
+
+	shortDescription = "Display one or more releases"
+	longDescription  = `Display one or more releases`
+
+	examples = `  # List all releases
+  kubectl gs get releases
+
+  # Get one specific release by its name
+  kubectl gs get release 1.4.2`
```
Comment (axbarsan) [inline cmd/get/releases/command.go:28]: This actually requires the `v` prefix to work. Or can we actually make it work both with and without the `v` prefix?
```suggestion
  kubectl gs get release v1.4.2`
```

Comment (AverageMarcus) [reply cmd/get/releases/command.go]: Fixed

File: cmd/get/releases/command.go (hunk 1 above)
Comment (AverageMarcus) [reply cmd/get/releases/command.go]: No real reason, updated to call it version instead.

**Review by axbarsan — APPROVED** [review]

**Pull request merged**